api_tests/reviews/mixins/filter_mixins.py

```diff
@@ -67,7 +68,7 @@ def allowed_providers(self, providers):
     @pytest.fixture()
     def expected_actions(self, all_actions, allowed_providers):
         provider_ids = set([p.id for p in allowed_providers])
-        return [l for l in all_actions if l.target.provider_id in provider_ids]
+        return [a for a in all_actions if a.target.provider_id in provider_ids]
```

Comment:
Would this be clearer if `a` --> `action`?

Reply:
I think I'm good with one-letter names in list comprehensions, but I agree with you everywhere else.